### ISSUE TITLE
Any chance you can help with converting this to block for me?

### ISSUE THREAD

**Issue by karl0ss** [jaybz/TP-HomeAssistant#31]
Great plugin, been using for awhile, But i am trying to get it to toggle a bunch of lights I have in a group, it would appear that we are unable to do that via the current way (only shows lights to toggle, not groups?)

So I was wanting to use the following block - 
![image](https://github.com/jaybz/TP-HomeAssistant/assets/2493260/2c5bbcf0-612c-4adc-b7b0-02207db1c7aa)

As it feels like this is exactly the way to just use the "home_assistant.core.toggle", but I can't work it out....here is a working yaml from HA

```
service: homeassistant.toggle
data: {}
target:
  entity_id: group.office
```

it feels like I should be able to do this based on your comment, but can't work it out..

thanks for response :)

**Comment by jaybz** [comment]
Try the following:
Domain: homeassistant
Service: toggle
Data: {"entity_id": "group.office"}

**Comment by karl0ss** [comment]
That worked perfectly, thank you so much! :)

**Comment by PhantasmNZ** [comment]
In case anyone else finds this and is confused on how to add other "data" parameters - for example in my case I wanted to specify  "brightness" - then the important things you need to know are that the Domain, Service and Data are simply part of the REST API (detailed here: https://developers.home-assistant.io/docs/api/rest/).  Now the API documentation didn't help me much with dimming - but it did lead me to this post: https://community.home-assistant.io/t/dimming-through-the-api/361922

so long story short, you can use services to turn on and choose brightness (and other things I assume) - in my case the working YAML was:

```
service: light.turn_on
target:
  entity_id: light.color_light_none
data:
  brightness_pct: 20
```
and the TP plugin block settings that corresponded to that were: 

Domain: light
Service: turn_on
Data: { "entity_id": "light.color_light_none", "brightness_pct":20}

Hope this helps a future visitor trying to work this out !!!
